PosteAPI/views.py

```diff
@@ -381,3 +402,19 @@ def get(self, request, pk):
             return Response({"success": True}, status=status.HTTP_200_OK)
         else:
             return Response({"success": False, "Error": "folder does not exist."}, status=status.HTTP_400_BAD_REQUEST)
+          
+class FolderDetail(APIView):
+    def get_object(self, pk):
```

Comment:
We should probably include `authentication_classes` and `permission_classes` (see line 220 under FolderAPI) to ensure only an authorized user can gain access to this data; it will require the request includes a token grabbed from the user in the request header (as "Authorization": "Token <tokenhere>")

This is stored client-side and has a method to retrieve; the update method under user demonstrates its usage